package.json

```
@@ -7,10 +7,10 @@
     "addons-linter": "bin/addons-linter"
   },
   "engines": {
-    "node": ">=16.0.0"
+    "node": ">=18.0.0"
```

Might be worth declaring a later version since that's what the tests will use. Node 18.20 is also the first version with `with` import assertions support, which could become useful.
```suggestion
    "node": ">=18.20"
```